.github/workflows/ci-integration.yml

```diff
@@ -18,17 +18,23 @@ jobs:
           - "datastax/lunastreaming:2.10_2.3"
           - "apachepulsar/pulsar:2.10.2"
         cassandra:
-          - cassandra_version: 3.0.32
-            ccm_version: 3.0.32
+          - cassandra_version: 3.0.27
+            ccm_version: 3.0.27
             is_dse: false
-          - cassandra_version: 3.11.19
-            ccm_version: 3.11.19
+          - cassandra_version: 3.11.13
+            ccm_version: 3.11.13
             is_dse: false
-          - cassandra_version: 4.0.19
-            ccm_version: 4.0.19
+          - cassandra_version: 4.0.4
+            ccm_version: 4.0.4
             is_dse: false
           - cassandra_version: dse-5.1
-            ccm_version: 5.1.48
+            ccm_version: 5.1.31
+            is_dse: true
+          - cassandra_version: dse-6.0
```

Comment:
DSE 6.0 and 6.7 are EOSL and we could remove/comment them here to save on some GH actions run time.